src/main/java/net/hypixel/nerdbot/generator/StringColorParser.java

```diff
@@ -9,310 +8,310 @@
 import java.util.ArrayList;
 
 public class StringColorParser {
+    private static final int maxLineLength = 38;
+    private final static MCColor[] colors = MCColor.VALUES;
+    private final static Stat[] stats = Stat.VALUES;
+    private final static Gemstone[] gemstones = Gemstone.VALUES;
+
+    // variables used to store the description
+    private final ArrayList<ArrayList<ColoredString>> parsedDescription;
+    private ArrayList<ColoredString> currentLine = new ArrayList<>();
+    private ColoredString currentString;
+
+    // variables for keeping track of line length and position
+    private int charIndex;
+    private int lineLength;
+
+    private String errorString;
+    private boolean successfullyParsed;
+
+    public StringColorParser() {
+        parsedDescription = new ArrayList<>();
+        currentString = new ColoredString();
+        currentString.setCurrentColor(MCColor.GRAY);
+
+        charIndex = 0;
```

Comment:
is there any reason these are being set in the constructor?

Reply:
Changed the constructor for the ColoredString at one point and obviously forgot to remove this